generators/app/templates/__tests__/screens/Login/testLoginScreen.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import CustomTextInput from '@components/CustomTextInput';
+import { shallow } from 'enzyme';
```

Comment:
Global imports first! Move this line above `import CustomTextInput from '@components/CustomTextInput';`